tensorflow/lite/micro/examples/person_detection/Makefile.inc

```diff
@@ -77,5 +77,10 @@ $(eval $(call microlite_test,detection_responder_test_int8,\
 $(DETECTION_RESPONDER_TEST_SRCS),$(DETECTION_RESPONDER_TEST_HDRS)))
 
 # Builds a standalone object recognition binary.
+ifneq ($(TARGET), esp)
 $(eval $(call microlite_test,person_detection_int8,\
 $(person_detection_SRCS),$(person_detection_HDRS)))
+else
+$(eval $(call microlite_test,person_detection,\
+$(person_detection_SRCS),$(person_detection_HDRS)))
+endif
```

Comment:
its not cleat to be why this is needed. I would prefer if `TARGET=esp` were to also with with person_detection_int8.

Reply:
@advaitjain 

Helper function uses example name as directory:

examples are:

```
$(PRJDIR)$(2)/esp-idf/%: tensorflow/lite/micro/examples/$(2)/esp/%

$(patsubst tensorflow/lite/micro/examples/$(2)/%,main/%,$(3) $(4) $(5) $(6))
```

This simply fails for person_detection example as now it doesn't match with generated example name.

For all other example it is the same, micro_speech goes with directory named micro_speech etc. 

I do not have a good solution to tackle this.

Reply:
ok, adding in @petewarden since he knows the details of project generation.

If this is an issue for esp, then I would expect the same issue to exist for say the Arduino and others and would like to fix it more generally.

Reply:
@advaitjain @petewarden got time to revisit this?